src/devices/Amg88xx/FrameRateBit.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the bit of the frame rate register (addr: 0x02)
+    /// </summary>
+    public enum FrameRateBit : byte
```

Comment:
So you may use this one for the FrameRate potential property and have something like
```csharp
public enum FrameRateBit : byte
{
   Rate10FramePerSecond = 0,
   Rate1FramePerSecond = 1,
}
```

Reply:
I agree, and for values either use what sensor uses internally or logically the same value (so Rate1FramePerSecond = 1, Rate10FramePerSecond = 10)

Reply:
@Ellerbach @krwq This enum does not define the frame rate itself. It defines the bit position in the corresponding register (as all the other xxxBit enums). So your suggestion doesn't work. However, it could be reasonable to define an additional enum for the actual frame rate setting, as you suggested.

Reply:
Should this enum be internal in that case?

Reply:
also since this has only a single value consider making it constant

Reply:
yes, this should be internal. I missed that.
I would keep it an enum for code consistency, though this is my taste.